composer.json

```diff
@@ -18,16 +18,16 @@
         }
     ],
     "require": {
-        "php": "^7.2|^8.0",
+        "php": "^7.4|^8.0",
         "ext-json": "*",
         "ext-openssl": "*",
-        "guzzlehttp/guzzle": "^6.5|^7.0",
-        "illuminate/contracts": "^7.9|^8.0|^9.0|^10.0",
-        "illuminate/database": "^7.9|^8.0|^9.0|^10.0",
-        "illuminate/http": "^7.9|^8.0|^9.0|^10.0",
-        "illuminate/routing": "^7.9|^8.0|^9.0|^10.0",
-        "illuminate/support": "^7.9|^8.0|^9.0|^10.0",
-        "illuminate/view": "^7.9|^8.0|^9.0|^10.0",
+        "guzzlehttp/guzzle": "^7.0",
```

Comment:
I would go with `^7.4.5` as the minimum (earlier versions have known security issues), or `^7.5.1` (has version constraints high enough to avoid all current known issues in peer dependencies).